resources-service/tests/service/server.ts

```diff
@@ -0,0 +1,33 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Out of curiosity: I'm assuming you see this test server setup deviating between resources-service and hrm-service and that is why it is duplicated. What are some ways that you expect them to deviate?

Reply:
They already deviate in terms of their default configurations, HRM passes in permissions stuff that resources doesn't (currently) need. They also run a different 'configureService' implementation. We could separate out the core bootstrapping logic because that is the same, but it's only a few lines, and it could well deviate, so I left it separate for now.